ghcup-prereleases-0.0.9.yaml

```diff
@@ -2837,6 +2838,123 @@ ghcupDownloads:
+            unknown_versioning: *ghc9102rc1-x86_64-deb10
+          Linux_Fedora:
+            '>= 33': &ghc9102rc1-x86_64-fedora33
+              dlHash: fbcb9309a0ab3b98e0626b96d3ebc34a65f909d040bed513dfb249801d223632
+              dlSubdir: ghc-9.10.1.20250417-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.1.20250417/ghc-9.10.1.20250417-x86_64-fedora33-linux.tar.xz
+          Linux_Mint:
+            < 20: &ghc9102rc1-x86_64-ubuntu1804
+              dlHash: 4761f8ce326bdfe5eb3953a5ffa2db096770b2b3ca5531142b33a6b3f386facf
+              dlSubdir: ghc-9.10.1.20250417-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.1.20250417/ghc-9.10.1.20250417-x86_64-ubuntu18_04-linux.tar.xz
+            '>= 20': &ghc9102rc1-x86_64-ubuntu2004
+              dlHash: 4c71f59165db6b13a1cd1a7824621363328bd8a25b00077e5e3c10d86afe637c
+              dlSubdir: ghc-9.10.1.20250417-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.1.20250417/ghc-9.10.1.20250417-x86_64-ubuntu20_04-linux.tar.xz
+            unknown_versioning: *ghc9102rc1-x86_64-ubuntu2004
+          Linux_RedHat:
+            < 9: &ghc9102rc1-x86_64-rocky8
+              dlHash: 5bf09938f196d19ce9fd4da472bc315386bbe02932d7baa28910651d1363f5f1
+              dlSubdir: ghc-9.10.1.20250417-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.1.20250417/ghc-9.10.1.20250417-x86_64-rocky8-linux.tar.xz
+            '>= 9': *ghc9102rc1-x86_64-fedora33
+            unknown_versioning: *ghc9102rc1-x86_64-fedora33
+          Linux_Ubuntu:
+            ( >= 16 && < 18 ): *ghc9102rc1-x86_64-deb9
+            ( >= 18 && < 19 ): *ghc9102rc1-x86_64-ubuntu1804
+            unknown_versioning: *ghc9102rc1-x86_64-ubuntu2004
+          Linux_UnknownLinux:
+            unknown_versioning: *ghc9102rc1-x86_64-rocky8
+          Windows:
+            unknown_versioning:
+              dlHash: 10aa40c687ad1d657b86332995699d8da09b51078f9754dc7ddea3f939add71c
+              dlSubdir: ghc-9.10.1.20250417-x86_64-unknown-mingw32
+              dlUri: https://downloads.haskell.org/~ghc/9.10.1.20250417/ghc-9.10.1.20250417-x86_64-unknown-mingw32.tar.xz
+        A_ARM64:
+          Darwin:
+            unknown_versioning:
+              dlHash: 455aa8c8c270bba894d4f649f440d8f9747be86133cdfffa718850b5a88a297d
+              dlSubdir: ghc-9.10.1.20250417-aarch64-apple-darwin
+              dlUri: https://downloads.haskell.org/~ghc/9.10.1.20250417/ghc-9.10.1.20250417-aarch64-apple-darwin.tar.xz
+          Linux_Alpine:
+            unknown_versioning:
+              dlHash: 7a1174d6beed044dc485686ff1f564594772e5339c6774a8dea1b206bef900b0
+              dlSubdir: ghc-9.10.1.20250417-aarch64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.1.20250417/ghc-9.10.1.20250417-x86_64-alpine3_18-linux.tar.xz
+          Linux_Debian:
+            ( >= 10 && < 11 ): &ghc9102rc1-x86_64-deb10
```

Comment:
This is a duplicated anchor, as CI clearly shows https://github.com/haskell/ghcup-metadata/actions/runs/14541665348/job/40800723322#step:9:64